code/modules/vehicles/mecha/mecha_defense.dm

```diff
@@ -204,78 +204,103 @@
 		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
 	return ..()
 
-/obj/vehicle/sealed/mecha/attackby(obj/item/W, mob/living/user, params)
+/obj/vehicle/sealed/mecha/attackby(obj/item/weapon, mob/living/user, params)
 	if(user.combat_mode)
 		return ..()
-	if(istype(W, /obj/item/mmi))
-		if(mmi_move_inside(W,user))
-			to_chat(user, span_notice("[src]-[W] interface initialized successfully."))
+	if(istype(weapon, /obj/item/mmi))
+		if(mmi_move_inside(weapon,user))
+			to_chat(user, span_notice("[src]-[weapon] interface initialized successfully."))
```

Comment:
A lot of these could probably be balloon alerts 🤔 
I am not requiring this as a blocker on merging though.